From 75895376001e904b7af62ad46dce64a93641e5e1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ji=C5=99=C3=AD=20Kone=C4=8Dn=C3=BD?=
 <DragonLichcz@gmail.com>
Date: Wed, 25 Nov 2015 21:27:17 +0100
Subject: [PATCH 3/3] Move build check_dd to /usr/libexec as says FHS

FHS (filesystem hierarchy standard) defines the /usr/share as
non-arch dependend directory.
So when we build the check_dd it should go to the /usr/libexec.
---
 Makefile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Makefile b/Makefile
index 54ca41f..a4fca7d 100644
--- a/Makefile
+++ b/Makefile
@@ -32,6 +32,8 @@ CFLAGS += -lGL -lX11
 
 sharedir := $(DESTDIR)$(PREFIX)/share
 bindir := $(DESTDIR)$(PREFIX)/bin
+execdir := $(DESTDIR)$(PREFIX)/libexec
+
 
 all: build
 
@@ -54,6 +56,7 @@ build:
 
 install:
 	install -d $(bindir)
+	install -d $(execdir)
 	install -d $(sharedir)/pixmaps
 	install -d $(sharedir)/applications
 	install -d $(sharedir)/playonlinux/bin
@@ -65,6 +68,7 @@ install:
 	cp ./etc/playonlinux16.png $(sharedir)/pixmaps/playonlinux16.png
 	cp ./etc/playonlinux32.png $(sharedir)/pixmaps/playonlinux32.png
 	cp ./bin/{playonlinux,playonlinux-pkg} $(bindir)/
+	cp ./bin/playonlinux-check_dd $(execdir)/
 	cp ./{playonlinux*,README.md,TRANSLATORS,CHANGELOG.md,LICENCE} $(sharedir)/playonlinux/
 	cp -R ./{bash,etc,lang,lib,plugins,python,resources,tests} $(sharedir)/playonlinux/
 
-- 
2.5.0

